app/components/Button/Button.css

```diff
@@ -57,38 +73,118 @@ html[data-theme='dark'] .button {
   margin-left: 8px;
 }
 
-.dark {
-  background: var(--color-red-3);
+.button.dark {
+  background: var(--color-red-2);
+  border: 1px solid rgba(--color-gray-4, 20%);
   color: var(--color-white);
-  border-color: var(--color-red-1);
+  box-shadow: none;
 
   &.fa {
     color: var(--color-white);
   }
 
-  &:active {
+  &:focus-visible {
+    box-shadow: 0 0 0 3px rgba(--color-red-3, 25%);
+  }
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-1);
+    border: 1px solid rgba(--color-gray-5, 20%);
+  }
+}
+
+html[data-theme='dark'] .button.dark {
+  background: var(--color-red-1);
+  border: 1px solid rgba(--color-light-gray-4, 20%);
+  color: var(--color-black);
+  box-shadow: none;
+
+  &:focus-visible {
+    box-shadow: 0 0 0 3px rgba(--color-red-5, 25%);
+  }
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-2);
+    border: 1px solid rgba(--color-light-gray-5, 20%);
+  }
+}
+
+.danger {
+  background: none;
+  box-shadow: none;
+  color: var(--color-red-2);
+
+  &:focus-visible {
+    box-shadow: 0 0 0 3px rgba(--color-red-3, 25%);
+  }
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-2);
+    box-shadow: none;
+    border: 1px solid var(--color-red-2);
     color: var(--color-white);
   }
 
   &:disabled {
-    color: rgba(var(--rgb-max), var(--rgb-max), var(--rgb-max), 30%);
+    background: none;
+    color: var(--color-red-2);
+    opacity: 0.5;
   }
 }
 
-.small {
-  padding: 10px 15px;
-  font-size: 0.8rem;
+html[data-theme='dark'] .danger {
+  background: none;
+  box-shadow: none;
+  color: var(--color-red-2);
+
+  &:focus-visible {
+    box-shadow: 0 0 0 3px rgba(--color-red-5, 25%);
+  }
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-2);
+    border: 1px solid var(--color-red-2);
+    color: var(--color-black);
+  }
+
+  &:disabled {
+    color: var(--color-red-2);
+  }
```

Comment:
Same here. Remove all the duplicate styles

Reply:
Sure, but if you don't want duplicates on the `&:hover`, you'll have to accept a bit of `!important` 😄 Or else, the hover effect cascaded from `.button` will take effect.

Reply:
I managed to remove all duplicates everywhere else though